### PR TITLE
New package: libpurple-mattermost-1.2_1

### PR DESCRIPTION
**Pull request by j0ack**
New package for libpurple-mattermost

### CODE REVIEW

File: srcpkgs/libpurple-mattermost/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+pkgname=libpurple-mattermost
+_git_name=purple-mattermost
+version=1.2
+revision=1
+wrksrc="$_git_name-$version"
+build_style=gnu-makefile
+makedepends="libpurple-devel json-glib-devel libglib-devel discount-devel base-devel"
+short_desc="Connects libpurple-based clients with Mattermost server"
+maintainer="TROUVERIE Joachim <joachim.trouverie@linoame.fr>"
+license="GPL-3.0"
```
Comment (Johnnynator) [inline srcpkgs/libpurple-mattermost/template]: pls use the SPDX identifier (with the `-or-later` or the `-only` postfix)

File: srcpkgs/libpurple-mattermost/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'libpurple-mattermost'
+pkgname=libpurple-mattermost
+_git_name=purple-mattermost
+version=1.2
+revision=1
+wrksrc="$_git_name-$version"
+build_style=gnu-makefile
+hostmakedepends="pkg-config"
+makedepends="libpurple-devel json-glib-devel libglib-devel discount-devel base-devel"
```
Comment (maxice8) [inline srcpkgs/libpurple-mattermost/template]: no need to have base-devel

File: srcpkgs/libpurple-mattermost/template [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'libpurple-mattermost'
+pkgname=libpurple-mattermost
+_git_name=purple-mattermost
```
Comment (maxice8) [inline srcpkgs/libpurple-mattermost/template]: i'd rather you just write it out

File: srcpkgs/libpurple-mattermost/template (hunk 2 above)
Comment (j0ack) [reply srcpkgs/libpurple-mattermost/template]: I thought `build-essential` which is in [the linux install guide](https://github.com/EionRobb/purple-mattermost/blob/master/INSTALL.md#linux-client-install-guide) was replaced in void by base-devel ?

File: srcpkgs/libpurple-mattermost/template (hunk 3 above)
Comment (j0ack) [reply srcpkgs/libpurple-mattermost/template]: Sorry but what do you mean here ?

Comment (j0ack) [reply srcpkgs/libpurple-mattermost/template]: Ok thanks

File: srcpkgs/libpurple-mattermost/template (hunk 2 above)
Comment (j0ack) [reply srcpkgs/libpurple-mattermost/template]: I get the answer on the irc channel, actually base-devel is already included into the builder no need to add it in makedepends

**Pull request merged**